share/arno-iptables-firewall/environment

```diff
@@ -320,18 +320,25 @@ ip4tables()
 {
   local result retval IFS=' '
 
-  result=`$IP4TABLES${IPTABLES_OPTIONS:+ $IPTABLES_OPTIONS} "$@" 2>&1`
+  exec 3>&1
+  err_result=`$IP4TABLES${IPTABLES_OPTIONS:+ $IPTABLES_OPTIONS} "$@" 2>&1 1>&3 |grep -v -e 'WARNING:.*match is obsolete' -e 'iptables-legacy tables present'`
   retval=$?
 
```

Comment:
I wonder if we want to add:
```
exec 3>&-
```
to close fd3 ?

Ref: https://tldp.org/LDP/abs/html/io-redirection.html

Reply:
Yes, we should. Forgot to cp that from aif-job-processor